.cargo/config.toml

```diff
@@ -9,4 +9,5 @@ rustflags = [
 
   # Unstable cfg options:
   # "--cfg", "skip_umdf_static_crt_check",
+  # "--cfg", "allow_stampinf_version_env_override"
```

Comment:
Since we're using the official env var of the tool here, i think its fine to not gate this behind unstable. Contingent on treating all the tool env vars in the same way as we treated `STAMPINF_VERSION` in this pr, i think removing the unstable cfg is fine. What are others' thoughts here @microsoft/windows-drivers-rs ?

Reply:
I agree. We were discussing the same thing yesterday. cfgs are a bit painful to set during cargo install as well. So just having the env var is enough.

Yes, we could generalize this scheme to other tools as well, but that overall UX design is yet to be done. Let's see how it plays out.

Reply:
Removed the cfg.